releasenotes/notes/create-diagnose-datadog-connectivity-subcommand-53403e7599a38084.yaml

```diff
@@ -0,0 +1,17 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+upgrade:
+  - |
+    ``diagnose`` command has now subcommands. The original ``diagnose``
+    behavior was transfered to ``diagnose metadata-availability`` subcommand.
```

Comment:
```suggestion
    The ``diagnose`` command has subcommands. The original ``diagnose``
    behavior was transferred to a ``diagnose metadata-availability`` subcommand.
```

Reply:
I removed this part as there is no breaking change anymore.